python/paddle/fluid/tests/unittests/ir/inference/test_mkldnn_conv_elementwise_add_fuse_pass.py

```diff
@@ -25,17 +25,120 @@
 import hypothesis.strategies as st
 
 
-class TestConvElementwiseAddMkldnnFusePass(PassAutoScanTest):
+# the two inputs of elementwise_add are tensor
+class TestConvElementwiseAddMkldnnFusePass1(PassAutoScanTest):
```

Comment:
```suggestion
class TestConvElementwiseAddMkldnnFusePass(PassAutoScanTest):
```

Reply:
This UT was added from https://github.com/PaddlePaddle/Paddle/pull/39654 . So I prefer any non-critical changes to left to original author.